samples/multiplatform/src/jvmTest/spek/src/test/kotlin/SmokeTest.kt

```diff
@@ -0,0 +1,31 @@
+package org.spekframework.spek2
+
+import ch.tutteli.atrium.api.fluent.en_GB.messageContains
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.fluent.en_GB.toThrow
+import ch.tutteli.atrium.api.verbs.expect
+import org.spekframework.spek2.style.specification.describe
+import java.lang.IllegalArgumentException
+
+object CalculatorTestBySpek : Spek({
```

Comment:
same here, it would be enough for me if you only use junit5 as test runner for JVM.

Reply:
for this test and mocha test above, I just copied from the ones already accepted, but I will see what I can do.